iot-e2e-tests/common/src/test/java/tests/integration/com/microsoft/azure/sdk/iot/iothub/telemetry/SendMessagesTests.java

```diff
@@ -67,6 +67,14 @@ public void sendMessages() throws Exception
         IotHubServicesCommon.sendMessages(testInstance.client, testInstance.protocol, NORMAL_MESSAGES_TO_SEND, RETRY_MILLISECONDS, SEND_TIMEOUT_MILLISECONDS, 0, null);
     }
 
+    @Test
+    public void sendBulkMessages() throws Exception
+    {
+        this.testInstance.setup();
+
+        IotHubServicesCommon.sendBulkMessages(testInstance.client, testInstance.protocol, NORMAL_MESSAGES_TO_SEND, RETRY_MILLISECONDS, SEND_TIMEOUT_MILLISECONDS, 0, null);
+    }
+
```

Comment:
I think a good CI-only test to complement the above test is to try a bulk message made up of a large number of small messages. What do you think?

Reply:
I like that. I will add a negative test as well to make sure the correct exception is thrown if the size of the message is above 255k

Reply:
it turns out we already have unit tests for the BatchHTTPmessages class the failure scenario